src/css/main.less

```diff
@@ -967,8 +976,69 @@ fieldset[disabled] .btn-info.active {
         font-weight: bold;
         font-size: @font-size-h3;
         line-height: @base-spacing-unit*2;
+        text-transform: uppercase;
         margin: @base-spacing-unit*2 0;
       }
+
+      ul.filters {
```

Comment:
We're nesting quite deeply here. Do we need the `ul`? Or to put this within `.tab-pane.active.flex-container`?